src/test/java/Homework22.java

```diff
@@ -0,0 +1,16 @@
+import Pages.HomePage;
+import Pages.LoginPage;
+import org.testng.annotations.Test;
+
+public class Homework22 extends BaseTest {
+
+    @Test(dataProvider = "CorrectLoginProviders", dataProviderClass = BaseTest.class)
+    public void LoginValidEmailValidPasswordTest(String email, String password) throws InterruptedException {
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        loginPage.login(email, password);
+        //Assert.assertTrue(homePage.getUserAvatar().isDisplayed());
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**

![Clean_Test](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/206856f2-8b07-480f-9db9-90a07bd9f4aa)